apps/src/templates/teacherNavigation/TeacherNavigationRouter.tsx

```diff
@@ -84,6 +85,20 @@ const TeacherNavigationRouter: React.FC<TeacherNavigationRouterProps> = ({
         : null
   );
 
+  const selectedSectionCourseName = useSelector(
```

Comment:
Can we get this from `selectedSection` instead of from a new selector? I think we should have it available below just from `selectedSection.courseDisplayName`